spec/rubocop/cop/rspec/contain_exactly_spec.rb

```diff
@@ -0,0 +1,32 @@
+describe RuboCop::Cop::RSpec::ContainExactly do
+  subject(:cop) { described_class.new }
+
+  it 'finds `match_array` instead of an instance double' do
```

Comment:
`instead of contain_exactly`. But I would reword it as 'flags `match_array` with only splat arguments'